tests/lib/rules/new-cap.js

```diff
@@ -60,8 +60,10 @@ ruleTester.run("new-cap", rule, {
 
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Bar"] }] },
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Foo.Bar"] }] },
+        { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptionPattern: "^Foo\.." }] },
```

Comment:
Could you add some tests for both new options with just an object, not object property?